src/reconcile/rec_visibility.c

```diff
@@ -368,25 +382,10 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
              * keep the same on-disk value but set the stop time pair to indicate that the validity
              * window ends when this tombstone started.
              */
-            upd_select->start_ts = vpack->start_ts;
```

Comment:
Why these assignments of start time pair is removed? Is it done somewhere else?

Reply:
Oh, those shouldn't be removed. my mistake.

Reply:
fixed